$NetBSD: patch-py-compile,v 1.1 2012/08/15 06:42:43 obache Exp $

* prevent to specify cfile for py_compile.compile to be PEP 3147-compliant.

--- py-compile.orig	2012-01-24 16:37:31.000000000 +0000
+++ py-compile
@@ -130,7 +130,7 @@ for file in files.split():
 	    continue
     sys.stdout.write(file)
     sys.stdout.flush()
-    py_compile.compile(filepath, filepath + 'c', path)
+    py_compile.compile(filepath, None, path)
 sys.stdout.write('\n')" || exit $?
 
 # this will fail for python < 1.5, but that doesn't matter ...
@@ -147,7 +147,7 @@ for file in files.split():
 	    continue
     sys.stdout.write(file)
     sys.stdout.flush()
-    py_compile.compile(filepath, filepath + 'o', path)
+    py_compile.compile(filepath, None, path)
 sys.stdout.write('\n')" 2>/dev/null || :
 
 # Local Variables:
